workspaces/jenkins/plugins/jenkins-backend/src/service/jenkinsApi.ts

```diff
@@ -344,15 +344,8 @@ export class JenkinsApiImpl {
 
   async getJobBuilds(jenkinsInfo: JenkinsInfo, jobFullName: string) {
     let jobName = jobFullName;
-
     if (jobFullName.includes('/')) {
-      const arr = jobFullName.split('/');
-      const multibranchJobName = arr.shift();
-      jobName = [
-        multibranchJobName,
-        'job',
-        encodeURIComponent(arr.join('/')),
-      ].join('/');
+      jobName = jobFullName.split('/').map(s => `${encodeURIComponent(s)}`).join('/job/');
```

Comment:
**nitpick (non-blocking)**: add the type for s:  `map(s: string) => ...`

Reply:
done